src/bridge/cockpitrouter.c

```diff
@@ -1132,6 +1217,8 @@ cockpit_router_add_bridge (CockpitRouter *self,
   g_return_if_fail (COCKPIT_IS_ROUTER (self));
   g_return_if_fail (config != NULL);
 
+  // TODO - reject dynamic privileged bridges
```

Comment:
Not sure what that means exactly, but should that block this PR?

Reply:
Bridges are usually spawned when the first channel is opened that requires them, and the command that is spawned can be constructed from options of that first channel.  The ssh bridges are an example.

This doesn't make sense for superuser bridges: We only want one of them, and there are no channel options when they are started.

Reply:
Also, the rest of the code assumes that `rule->user_data` is a CockpitPeer.  I have implemented this now.